### DISCUSSION TITLE
Need help to use HA blueprint as software connector

### DISCUSSION THREAD

**Discussion by BDVGitHub** [davidusb-geek/emhass#402] [Q&A] [answered]
Is there annyone who is familiar with setting up a blueprint in Home Assistant? As discussed here https://github.com/davidusb-geek/emhass/issues/394, we would would like to use an blueprint to use as an software connector so that for example [Nord Pool](https://www.home-assistant.io/integrations/nordpool/) values can be used in Emhass without the need for the user the use of an complex shell command.

I had an look at the forum and found [About blueprints](https://community.home-assistant.io/t/about-blueprints/253788), there it stand that the blueprint must be placed in [blueprints exchange](https://community.home-assistant.io/c/blueprints-exchange/53) on the HA forum. To make the blueprint easy to find on the forum, I was thinking to have the name of the blueprint start with Emhass then the name of the integration and then connector, this way we can keep the name of the blueprint uniform and short should multiple blueprints be used. So e.g. Emhass Nord Pool connector, or what do you think?

So that we can use something like the below in place of the long shell command currently used to pass energy prices to Emhass.
`shell_command:
  dayahead_optim: "curl -i -H \"Content-Type: application/json\" -X POST -d '{\"load_cost_forecast\":{{states('sensor.load_cost_forecast_list')}}}' http://localhost:5000/action/dayahead-optim"`

Verder weet ik niet goed hoe de automation in HA kan weten welke frequentie gebruikt is in Emhass of dien je gewoon de prijzen per uur door te geven aan Emhass?

@purcell-lab Would you happen to have used blueprints in Home Assistant yet?

**Comment by purcell-lab** [comment]
Some blueprint action on discord:

https://discord.com/channels/1317814601466773505/1317982438005411902/1326316496967700700

**Reply by BDVGitHub** [reply, marked as answer]
Found the direct link to the EMHASS BluePrint development
https://discord.com/channels/1317814601466773505/1326316496967700700
